source/includes/aggregation.php

```diff
@@ -0,0 +1,40 @@
+<?php
+require 'vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+$collection = $client->sample_restaurants->restaurants;
+
+// Retrieves documents with a cuisine value of "Bakery", groups them by "borough", and
+// counts each borough's matching documents
+// start-match-group
+$pipeline = [
+    ['$match' => ['cuisine' => 'Bakery']],
+    ['$group' => ['_id' => '$borough', 'count' => ['$sum' => 1]]],
+];
+
+$cursor = $collection->aggregate($pipeline);
+
+foreach ($cursor as $doc) {
+    echo json_encode($doc) . PHP_EOL;
```

Comment:
`echo` supports any number of comma-delimited arguments, so you could also consider:

```suggestion
    echo json_encode($doc), PHP_EOL;
```

This avoids string concatenation and would be slightly more efficient.

Also applies to `echo` for the explain output. I might have missed this in previous PRs, so up to you if you'd like to address. OK to leave as-is if a bunch of docs pages are already doing `. PHP_EOL`.

Reply:
I'll change to a comma here and add that to the cleanup tasks